### ISSUE TITLE
regarding Objaverse-N200

### ISSUE THREAD

**Issue by eriche2016** [ganchaofan0000/DAC#1]
Hi, nice work, and thanks for sharing the code. 
However, the code is not complete. 
I am wondering if you could share the Objaverse-N200 dataset and its code. 
Looking forward to your reply.

**Comment by ganchaofan0000** [comment]
Thank you for your attention!

Our dataset, Objaverse-N200, consists of two components: 1) preprocessed object features extracted by [Openshape](https://github.com/Colin97/OpenShape_code) and 2) our assigned noisy category labels. Specifically, the Objaverse feature data can be found in [objaverse.pt](https://huggingface.co/datasets/OpenShape/openshape-objaverse-embeddings/blob/main/objaverse.pt). 
For the noisy data annotation, we utilized a pre-trained model [Uni3D](https://github.com/baaivision/Uni3D) to assign categories to these objects. We provide the data annotation in  ./data/train_200.json of this project.

For the code of Objaverse-N200 dataset, we will release it as soon as possible.

Thank you for your attention!

**Comment by ganchaofan0000** [comment]
> Hi, nice work, and thanks for sharing the code. However, the code is not complete. I am wondering if you could share the Objaverse-N200 dataset and its code. Looking forward to your reply.

Thank you for your attention!

Our dataset, Objaverse-N200, consists of two components: 1) preprocessed object features extracted by [Openshape](https://github.com/Colin97/OpenShape_code) and 2) our assigned noisy category labels. Specifically, the Objaverse feature data can be found in [objaverse.pt](https://huggingface.co/datasets/OpenShape/openshape-objaverse-embeddings/blob/main/objaverse.pt). 
For the noisy data annotation, we utilized a pre-trained model [Uni3D](https://github.com/baaivision/Uni3D) to assign categories to these objects. We provide the data annotation in  ./data/train_200.json of this project.

For the code of Objaverse-N200 dataset, we will release it as soon as possible.

Thank you for your attention!

**Comment by XinhanWang** [comment]
Hi @ganchaofan0000,

Thank you for sharing your work on DAC! It looks very interesting.

I’m particularly interested in the Objaverse-N200 dataset mentioned in your paper and codebase. Would it be possible for you to share the training code (and/or data preprocessing pipeline) used with Objaverse-N200?

That would greatly help the community to reproduce and build upon your results.

Looking forward to your reply!

Best regards!
